File: Baekjoon/11116.c
#include <stdio.h>

int main(){
    int testcase;
    scanf("%d", &testcase);
    for(int i=0; i<testcase; i++){
        int measure=0;
        scanf("%d", &measure);
        int left[200], right[200];
        int l2r = 0;
        for(int j=0;j<measure;j++){
            scanf("%d", &left[j]);
        }
        for(int j=0;j<measure;j++){
            scanf("%d", &right[j]);
        }
        for(int j=0;j<measure;j++){
            if(left[j] < right[j]) l2r++;
        }
        printf("%d", l2r/2);
    }
    return 0;
}

File: Baekjoon/11116.py
testcase = int(input())
for i in range(testcase):
    measure_num = int(input())
    left=[]
    right=[]
    res = 0
    l2r = 0
    # for j in range(measure_num):
    #     left = list(map(int, input().split()))
    left = list(map(int, input().split()))

    # for j in range(measure_num):
    #     right = list(map(int, input().split()))
    right = list(map(int, input().split()))
        
    while left:
        leftone = left.pop()
        if leftone + 1000 in right:
            res += 1

        if res == 2:
            res = 0
            l2r += 1

    print(l2r)